### PR TITLE
Add a JUnit tag for @QuarkusTest so that we can exclude them easily

### PR DESCRIPTION
**Pull request by gsmet**
If you mix tests in your project, you will need to exclude the
@QuarkusTests so we'd better include a tag for it.

### CODE REVIEW

**Comment by famod** [comment]
Good idea!
First I thought this would be a breaking change but it isn't; you don't need to list all tags of a test to have it executed, just one.
So it should co-exist nicely with project-specific tags.

CI is broken though, some problem with the cached Maven repo archive.

**Comment by famod** [comment]
Btw, might be worth adding some documentaion one-liner?

**Review by rsvoboda — APPROVED** [review]

**Comment by gsmet** [comment]
> Btw, might be worth adding some documentation one-liner?

Grmbl. Yeah, OK, it could use some documentation. I'll do that later today.

**Comment by rsvoboda** [comment]
documentation reminder ^^^ @gsmet ;)

**Comment by famod** [comment]
Can we also backport this? Even to 2.1?

**Comment by gsmet** [comment]
I pushed an update with the doc.

File: docs/src/main/asciidoc/getting-started-testing.adoc [hunk 1]
```diff
@@ -1151,6 +1151,51 @@ As a test annotated with `@QuarkusIntegrationTest` tests the result of the build
 These tests will **not** work if run in the same phase as `@QuarkusTest` as Quarkus has not yet created the final artifact.
 ====
 
+== Mixing `@QuarkusTest` with other type of tests
+
+It is not possible to execute a `@QuarkusTest` test with other types of tests in a single execution run (e.g. a Maven Surefire Plugin execution).
```
Comment (famod) [inline docs/src/main/asciidoc/getting-started-testing.adoc:1156]: That sentence alone reads as if it's not possible to mix with other, non-`@Quarkus*` tests. But as the next sentence is explicitely listing the conflicting `@Quarkus*` flavors, I think this is clear enough.

**Review by famod — APPROVED** [review]

File: docs/src/main/asciidoc/getting-started-testing.adoc (hunk 1 above)
Comment (gsmet) [reply docs/src/main/asciidoc/getting-started-testing.adoc]: I adjusted things a bit.

Comment (famod) [reply docs/src/main/asciidoc/getting-started-testing.adoc]: Perfection!

**Comment by rsvoboda** [comment]
+1 for the docs update

**Comment by gsmet** [comment]
@glefloch @famod these Gradle failures look new?

**Comment by glefloch** [comment]
Right but it looks similar to the flaky windows failures. Investigating on it.

**Pull request merged**